requirements.txt

```diff
@@ -4,15 +4,15 @@ py2app==0.28.8
 pyobjc==10.3.1
 corefoundationasyncio==0.0.1
 cocoa-vanilla==0.6.0
-blackrenderer==0.6.0
-fonttools[woff,lxml,unicode,ufo,type1]==4.53.1
+### blackrenderer==0.6.0 # moved to setup.py
```

Comment:
These should stay, as they get updated by dependabot. Also, in setup.py you should ideally not pin dependencies, although you are right they should be mentioned (but either without version, or with a minimal version).

Reply:
Totally makes sense. I did keep the pinned `==` for `python-bidi` since there’s the issue of the interface change there, does that work?